src/Avalonia.Controls/PullToRefresh/RefreshContainer.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.PullToRefresh;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+
+namespace Avalonia.Controls
+{
+    public class RefreshContainer : ContentControl
+    {
+        internal const int DefaultPullDimensionSize = 100;
+
+        private readonly bool _hasDefaultRefreshInfoProviderAdapter;
```

Comment:
should this property be always true?

Reply:
No. It should be false if the user set's a custom adapter. This will be updated.